### ISSUE TITLE
Permissions

### ISSUE THREAD

**Issue by valadas** [DNNCommunity/Dnn.StructuredContent#4]
I am just creating this issue to not forget about it but it's not a very fleshed out issue ready to tackle. I just want to try and collect everything we need to make this project in a state to allow it to be (v1 worthy) 😄 

We need to handle permissions for:
- Content Editing
- Web API Access
- UI views (visualizer?)

We can break this down in smaller more manageable items, but just wanted to dump the idea here for now.

**Comment by WillStrohl** [comment]
Yeah, this probably needs to be broken out into different issues later.
